internal/osbuild/zipl_stage.go

```diff
@@ -0,0 +1,25 @@
+package osbuild
+
+// The ZiplStageOptions describe how to create zipl stage
+//
+// The only configuration option available is a boot timeout and it is optional
+type ZiplStageOptions struct {
+	Timeout int `json:"timeout,omitempty"`
+}
+
+func (ZiplStageOptions) isStageOptions() {}
+
+// NewZiplStageOptions creates a new ZiplStageOptions object with no timeout
```

Comment:
I don't think this constructor is necessary, mostly we instantiate the structs directly. But that's a nitpick.